script.js

```diff
@@ -0,0 +1,190 @@
+document.addEventListener('DOMContentLoaded',function(event){
```

Comment:
You don't really need an event here. Cos you're not doing anything to it.
You can just write, ('DOMcontentLoaded', function() {